src/ui/public/filter_bar/filter_bar.html

```diff
@@ -51,75 +54,68 @@
       <a class="action filter-remove" ng-click="removeFilter(filter)">
         <i class="fa fa-fw fa-trash" data-test-subj="removeFilter-{{ filter.meta.key }}"></i>
       </a>
-      <a class="action filter-edit" ng-click="startEditingFilter(filter)">
+      <a class="action filter-edit" ng-click="editFilter(filter)">
         <i class="fa fa-fw fa-edit"></i>
       </a>
     </div>
   </div>
 
-  <div class="filter-edit-container" ng-if="editingFilter">
-    <form role="form" name="editFilterForm" ng-submit="editDone()">
-      <div class="form-group filter-edit-alias">
-        <label>Filter Alias
-          <small>(optional)</small>
-        </label>
-        <input
-          ng-model="editingFilter.alias"
-          type="text"
-          class="form-control">
-      </div>
-      <div
-        json-input
-        require-keys="true"
-        ui-ace="{
-          mode: 'json',
-          onLoad: aceLoaded
-        }"
-        ng-model="editingFilter.model"
-      ></div>
-
-      <div class="form-group">
-        <button
-          class="kuiButton kuiButton--hollow"
-          ng-click="stopEditingFilter()"
-        >
-          Cancel
-        </button>
-
-        <button
-          type="submit"
-          class="kuiButton kuiButton--primary"
-          ng-disabled="editFilterForm.$invalid"
-        >
-          Done
-        </button>
-
-        <small ng-show="editFilterForm.$invalid">
-          Could not parse JSON input
-        </small>
-      </div>
-    </form>
+  <div
+    class="filter-link"
+    ng-show="showAddFilterButton()"
+  >
+    <div class="filter-description small">
+      <a ng-click="addFilter()">
```

Comment:
You can add `kbn-accessible-click` here to make this button keyboard-accessible.